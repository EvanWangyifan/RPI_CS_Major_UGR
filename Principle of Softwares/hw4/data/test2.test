m
r
Polytechnic Residence Commons
Blitman Residence Commons
r
81
20
r
100
300
r
Invalid1
Invalid2