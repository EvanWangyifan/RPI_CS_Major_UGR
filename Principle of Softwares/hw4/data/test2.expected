commands available:
b: list all buildings in (name,id) format in lexicographic order.
r: input two building and get a shortest path.
q: quit the program.
m: list the menu.
First building id/name, followed by Enter: Second building id/name, followed by Enter: There is no path from Polytechnic Residence Commons to Blitman Residence Commons.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from RPI Ambulance to Cogswell Laboratory:
	Walk North to (Intersection 99)
	Walk NorthWest to (Cogswell Laboratory)
Total distance: 143.488 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [100]
Unknown building: [300]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Invalid1]
Unknown building: [Invalid2]